students/6/main.js

```diff
@@ -14,35 +14,46 @@ const ex2 = () => {
 }
 
 const ex3 = () => {
+    const array = ['this', 'is', 'a', 'test', 'happy'];
+    console.log(shortestString(array));
+}
+
+
+
+
+const shortestString = (array) => {
+    let retval = array.sort((a,b) => b.length - a.length);
+       return retval[array.length - 1];
```

Comment:
Can you fix this indentation issue?  This will be a big issue when we start Python.